vsintegration/src/FSharp.Editor/LanguageService.fs

```diff
@@ -207,13 +207,15 @@ type internal FSharpLanguageService(package : FSharpPackage) as this =
     let projectInfoManager = this.Package.ComponentModel.DefaultExportProvider.GetExport<ProjectInfoManager>().Value
 
     /// Sync the information for the project 
-    member this.SyncProject(project: AbstractProject, projectContext: IWorkspaceProjectContext, site: IProjectSite) =
+    member this.SyncProject(project: AbstractProject, projectContext: IWorkspaceProjectContext, site: IProjectSite, forceUpdate) =
```

Comment:
`SyncProject` is a class member with two usages at the same file. At the both call sites, I have used named parameters for `forceUpdate`. I don't think changing bool to two-case enums would improve readability here.